packages/loaders/prisma/src/prisma-yml/PrismaDefinition.test.ts

```diff
@@ -129,7 +129,8 @@ type User @model {
     const { definition, env } = makeDefinition(yml, datamodel, {});
     const envPath = path.join(definition.definitionDir, '.env');
 
-    fs.outputFileSync(envPath, `MY_DOT_ENV_SECRET=this-is-very-secret,and-comma,seperated`);
+    fs.mkdirSync(path.dirname(envPath), { recursive: true });
```

Comment:
Are you sure this does the same job?

Reply:
From https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputFile-sync.md

> Almost the same as writeFileSync (i.e. it overwrites), except that if the parent directory does not exist, it's created

Yes, mkdir with recursive flag creates directory if it does not exist